ca/certificate-authority.go

```diff
@@ -156,7 +121,15 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		return nil, err
 	}
 
-	pa := policy.NewPolicyAuthorityImpl()
+	dbMap, err := sa.NewDbMap(paConfig.DBConnect)
+	if err != nil {
+		return nil, err
+	}
+
+	pa, err := policy.NewPolicyAuthorityImpl(dbMap, paConfig.EnforcePolicyWhitelist)
```

Comment:
This is necessary for MySQL cleanup in CA and RA tests where a 'real' PADB is required.